assets/app/scripts/services/navigate.js

```diff
@@ -12,7 +12,7 @@ angular.module("openshiftConsole")
        */
       toErrorPage: function(message, errorCode) {
         var redirect = URI('error').query({
-          error_description: message,
+          'error_description': message,
```

Comment:
it was unhappy about this particular key, but not an unquoted "error" key? weird